.github/workflows/elixir.yml

```diff
@@ -7,57 +7,56 @@ on:
     branches: [master]
 
 jobs:
-
   build:
     name: Build and test
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        elixir_version: ['1.7.4', '1.8.2', '1.9.4', '1.10.3', '1.11.3']
-        otp_version: ['20.3.8', '21.2', '23.0.2', '22.3.4']
+        elixir_version: ["1.7.4", "1.8.2", "1.9.4", "1.10.3", "1.11.3", "1.12.3", "1.13.4", "1.14.5", "1.15.8", "1.16.3", "1.17.3", "1.18.3"]
+        otp_version: ["20.3.8", "21.2", "23.0.2", "22.3.4", "23.3.4", "24.3.4", "25.3.2", "26.2.5", "27.3"]
         exclude:
           # Unsupported
-          - elixir_version: '1.11.3'
-            otp_version: '20.3.8'
+          - elixir_version: "1.11.3"
+            otp_version: "20.3.8"
           # Unsupported
-          - elixir_version: '1.10.3'
-            otp_version: '20.3.8'
+          - elixir_version: "1.10.3"
+            otp_version: "20.3.8"
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Set up Elixir ${{ matrix.elixir_version }} on OTP ${{ matrix.otp_version }}
-      uses: actions/setup-elixir@v1
-      with:
-        elixir-version: ${{ matrix.elixir_version }}
-        otp-version: ${{ matrix.otp_version }}
-    - name: Restore dependencies cache
-      uses: actions/cache@v2
-      with:
-        path: deps
-        key: ${{ runner.os }}-mix-${{ matrix.otp_version }}-${{ matrix.elixir_version }}-${{ hashFiles('**/mix.lock') }}
-        restore-keys: ${{ runner.os }}-mix-${{ matrix.otp_version }}-${{ matrix.elixir_version }}-
-    - name: Install dependencies
-      run: mix deps.get --only test
-    - name: Run mix test
-      run: MIX_ENV=test mix test
+      - uses: actions/checkout@v2
+      - name: Set up Elixir ${{ matrix.elixir_version }} on OTP ${{ matrix.otp_version }}
+        uses: actions/setup-elixir@v1
```

Comment:
I think we should migrate this to erlef/setup-beam, just like prometheus.erl does, and up the versions of erlang and elixir to some latest ones 🤔

Reply:
done